src/arrays/ladderops.jl

```diff
@@ -37,7 +37,16 @@ function momentumop(n::Int)
     return scale(im/sqrt(2.), cop-cop')
 end
 
+squeeze_construct(a::AbstractQuMatrix,b::AbstractQuMatrix, z::Union(Complex64, Float64)) = 0.5*(z'*(a*b)-z*(a'*b'))
+squeezing(a::AbstractQuMatrix, b::AbstractQuMatrix, z::Union(Complex64, Float64)) = expm(squeeze_construct(a,b,z))
+squeeze(n::Int,z::Union(Complex64, Float64)) = squeezing(lowerop(n),lowerop(n),z)
+
+displaceop(n::Int,alpha::Union(Complex64, Float64)) = expm(alpha*lowerop(n)'-alpha'*lowerop(n))
```

Comment:
I think it is better to use `raiseop(n)` instead of `lowerop(n)!`, which makes an additional copy. In fact it would be even better to define
```
function displacementop( nb::Int, alpha::Number )
	ad = scale!(alpha, raiseop( nb ))
	# displacement to location alpha
	return expm( ad - ad')
end
```